lms/djangoapps/grades/api/v0/urls.py

```diff
@@ -0,0 +1,20 @@
+""" Courseware API v0 URLs. """
+
+from django.conf import settings
+from django.conf.urls import (
+    patterns,
+    url,
+)
+
+from lms.djangoapps.grades.api.v0 import views
+
+
+urlpatterns = patterns(
+    '',
+    url(
+        r'^course_grade/{course_id}/$'.format(
```

Comment:
Can you add `users` to the end of this URL?  For 2 reasons:
1. This endpoint is returning the course grade for users.  As opposed to other course grade related information such as grading policies, etc.
2. Soon, we will want this endpoint to return the grades for _all_ the users in the course.  Until then, the `username` URL parameter serves as a filter for which grades it returns.

Given this, we'll also need to update the response of your endpoint to return a `list` of dicts instead of just a single `dict`.  For now, it would be a `list` of only one `dict` element.